data/gqlHelper.js.njk

```diff
@@ -0,0 +1,129 @@
+//
+// !!! DO NOT EDIT !!!!
+//
+// This file was auto-generated by: node data/gqlSetup.js 
+// using the Nunjucks template at data/gqlHelper.js.njx
```

Comment:
didn't see this file?

Reply:
That's the template - the generated gqlHelper.js is the thing that should not be edited ...

Reply:
`gqlHelper.js.njx` typo? actually njk?